src/uikit.cs

```diff
@@ -2063,6 +2063,101 @@ interface UIObjectRestoration {
 #endif
 	}
 
+	interface IUIViewAnimating {}
+
+	[iOS(10,0)]
+	[Protocol]
+	[BaseType (typeof(NSObject))]
+	interface UIViewAnimating
+	{
+		[Abstract]
+		[Export ("state")]
+		UIViewAnimatingState State { get; }
+
+		[Abstract]
+		[Export ("running")]
+		bool Running { [Bind ("isRunning")] get; }
+
+		[Abstract]
+		[Export ("reversed")]
+		bool Reversed { [Bind ("isReversed")] get; set; }
+
+		[Abstract]
+		[Export ("fractionComplete")]
+		nfloat FractionComplete { get; set; }
+
+		[Abstract]
+		[Export ("startAnimation")]
+		void StartAnimation ();
+
+		[Abstract]
+		[Export ("pauseAnimation")]
+		void PauseAnimation ();
+
+		[Abstract]
+		[Export ("stopAnimation:")]
+		void StopAnimation (bool withoutFinishing);
+
+		[Abstract]
+		[Export ("finishAnimationAtPosition:")]
+		void FinishAnimationAtPosition (UIViewAnimatingPosition finalPosition);
+	}
+
+	interface IUIViewImplicitlyAnimating {}
+	[iOS(10,0)]
+	[Protocol, Model]
```

Comment:
is the `[Model]` needed ? it looks like something you'll implement in a type (not subclass)
if needed then it needs ` `[BaseType]`